core-bundle/contao/library/Contao/Database/Statement.php

```diff
@@ -256,6 +256,11 @@ public function query($strQuery='', array $arrParams = array(), array $arrTypes
 			$arrParams[$key] = serialize($varParam);
 		}
 
+		// null types are deprecated in doctrine/dbal 3.x
+		$arrTypes = array_map(static function ($type) {
+			return null === $type ? ParameterType::STRING : $type;
+		}, $arrTypes);
+
```

Comment:
This can be removed as all `null` values are already set to `ParameterType::STRING` on line 248.
_(Because #7126 was merged since this PR started)_